teleop:
  joy_pitch: 1
  joy_yaw: 0
  joy_forward: 2
  joy_updown: 4
